host_core/lib/host_core/control_interface/server.ex

```diff
@@ -328,14 +317,14 @@ defmodule HostCore.ControlInterface.Server do
                 start_provider_from_bindle(
                   start_provider_command["provider_ref"],
                   start_provider_command["link_name"],
-                  Map.get(start_provider_command, "configuration", ""),
+                  Map.get(start_provider_command, "configuration") || "",
```

Comment:
These two are functionally equivalent, right? Just wondering the motivation for the change

Reply:
I can't remember 🤔 I'll revert this and the change below.